hivemind/p2p/p2p_daemon.py

```diff
@@ -175,20 +193,32 @@ async def receive_protobuf(in_proto_type, reader):
         protobuf.ParseFromString(await P2P.receive_raw_data(reader))
         return protobuf
 
+    @staticmethod
+    async def receive_protobuf_with_error(in_proto_type, reader):
```

Comment:
```suggestion
    async def receive_protobuf_or_error(in_proto_type, reader):
```

The current name may be misleading since the received protobuf does not always contain an error.